launchable/commands/subset.py

```diff
@@ -347,8 +332,8 @@ def run(self):
 
                     res.raise_for_status()
 
-                    output = res.json()["testPaths"]
-                    rests = res.json()["rest"]
+                    original_subset = res.json().get("testPaths", [])
+                    original_rests = res.json().get("rest", [])
```

Comment:
[question]
Just to be sure, is it correct the default value is [] if the response is empty?

```python
original_subset = res.json().get("testPaths", original_subset)
original_rests = res.json().get("rest", original_subset)
```